src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+@import '/src/styles/variables';
+
+html {
+  font-family: Roboto, serif;
+}
+
 body {
-  margin: 0;
+  margin: $margin;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px;
+    height: 166px;
+    background-position: 19px 32px;
+  }
+
+  &__apple {
+    margin: $margin;
+    padding: 40px 16px 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black;
```

Comment:
The variable `$black` is used here, but it's not defined within this file. Ensure that it is defined in the imported `variables` file or elsewhere in your project.